test/jdk/javax/imageio/plugins/jpeg/JpegWriterWriteNonOpaqueIndexColorModelTest.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/**
+ * @test
+ * @bug 8351108
+ * @summary This test verifies that attempting to write a JPEG using
+ *          ImageIO.write(..) fails by returning `false` for
+ *          translucent IndexColorModels.
+ */
+
+import javax.imageio.ImageIO;
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.awt.image.IndexColorModel;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+public class JpegWriterWriteNonOpaqueIndexColorModelTest {
+    public static void main(String[] args) throws IOException {
+        boolean b1 = testJpegWriter(Transparency.OPAQUE, "OPAQUE", true);
+        boolean b2 = testJpegWriter(Transparency.BITMASK, "BITMASK", false);
+        boolean b3 = testJpegWriter(Transparency.TRANSLUCENT, "TRANSLUCENT", false);
+        if (!(b1 && b2 && b3))
+            throw new Error("Test failed");
+    }
+
+    private static boolean testJpegWriter(int imageType, String name, boolean expectedWriteReturnValue) {
+        System.out.println();
+        System.out.println("TESTING " + name);
+        try {
+            byte[] gray = new byte[256];
+            for (int a = 0; a < gray.length; a++) {
+                gray[a] = (byte) a;
+            }
+
+            byte[] alpha = new byte[256];
+            if (imageType == Transparency.OPAQUE || imageType == Transparency.BITMASK) {
+                for (int a = 0; a < alpha.length; a++) {
+                    alpha[a] = -1;
+                }
+                if (imageType == Transparency.BITMASK)
+                    alpha[0] = 0;
```

Comment:
```suggestion
                if (imageType == Transparency.BITMASK) {
                    alpha[0] = 0;
                }
```
It's recommended to always use braces even when there's only one statement in the block.